localisation/00_wc_random_artifact_names.csv

```diff
@@ -21,60 +37,60 @@ artifact_rare_origin;Fables tell that [GetArtifactRareOrigin].;Des fables disent
+;;;;;;;;;;;;;;x
+artifact_legendary_origin_1;[GetGrammaticalNumberThisThese] [This.GetName] belongs to those who are dead. All who have owned [GetGrammaticalNumberItThem] have died, and all who wear [GetGrammaticalNumberItThem] yet shall follow;ce [This.GetName] appartient � ceux qui sont morts. Tous ceux qui l'ont port� sont morts, et tous ceux qui le portent encore suivront;;;;;;;;;;;;x
+artifact_legendary_origin_2;many a hero has possessed [GetGrammaticalNumberThisThese] [This.GetName] through many an age, and the tales sung of [GetGrammaticalNumberItThem] can be heard from one end of the world to the other;de nombreux h�ros ont mani� ce [This.GetName] � travers les �ges, et les r�cits qu'on en fait peuvent �tre entendus d'un bout � l'autre du monde;;;;;;;;;;;;x
+artifact_legendary_origin_3;[GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberWasWere] crafted with a technique millennia beyond its time. Few craftsmen have been able to replicate the quality;ce [This.GetName] a �t� con�u avec une technique mill�naire au-del� de son temps. Peu d'artisans ont �t� capables de reproduire sa qualit�;;;;;;;;;;;;x
+artifact_legendary_origin_4;all who chance to behold [GetGrammaticalNumberThisThese] [This.GetName], desire [GetGrammaticalNumberItThem]. Those who possess [GetGrammaticalNumberItThem] must shed blood to keep [GetGrammaticalNumberItThem]. And those who deem themselves above [GetGrammaticalNumberItsTheir] allure are the most endangered by [GetGrammaticalNumberItThem];tous ceux qui ont la chance de voir ce [This.GetName] le d�sirent. Ceux qui le poss�dent doivent verser du sang pour le garder, et ceux qui se consid�rent au-dessus de son attrait sont les plus menac�s par lui;;;;;;;;;;;;x
+artifact_legendary_origin_5;[GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberWasWere] destroyed aeons ago..;ce [This.GetName] a �t� d�truit il y a des si�cles..;;;;;;;;;;;;x
+artifact_legendary_origin_6;the very sight of [GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberWasWere] enough to settle disputes that would otherwise lead to war;la seule vue de ce [This.GetName] a suffi � r�gler des diff�rends qui auraient autrement men� � la guerre;;;;;;;;;;;;x
+artifact_legendary_origin_7;no mortal hands crafted [GetGrammaticalNumberThisThese] [This.GetName];aucun mortel n'a fabriqu� ce [This.GetName];;;;;;;;;;;;x
+artifact_legendary_origin_8;[GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberHasHave] existed since before time began;ce [This.GetName] existe depuis bien avant le d�but des temps;;;;;;;;;;;;x
+artifact_legendary_origin_9;[GetGrammaticalNumberThisThese] [This.GetName] holds the secret to existence for those who can decipher [GetGrammaticalNumberItsTheir] intricate design;ce [This.GetName] d�tient le secret de l'existence pour ceux qui peuvent d�chiffrer sa conception complexe;;;;;;;;;;;;x
```

Comment:
Changed `existence` to `eternal life`, because I think it fits better. It could also be changed from `the secret to existence` to `the secret of life`.
```suggestion
artifact_legendary_origin_9;[GetGrammaticalNumberThisThese] [This.GetName] holds the secret to eternal life for those who can decipher [GetGrammaticalNumberItsTheir] intricate design;ce [This.GetName] d�tient le secret de l'existence pour ceux qui peuvent d�chiffrer sa conception complexe;;;;;;;;;;;;x
```

Reply:
Then it will kinda conflict with `of Everlasting Life` affix.

Reply:
Point. In that case, existence works fine.